packages/models-library/src/models_library/services.py

```diff
@@ -185,6 +220,27 @@ class ServiceOutput(ServiceProperty):
         deprecated=True,
     )
 
+    class Config(ServiceProperty.Config):
+        schema_extra = {
+            "examples": [
+                # version 1
+                {
+                    "displayOrder": 2,
+                    "label": "Time Slept",
+                    "description": "Time the service waited before completion",
+                    "type": "number",
+                },
+            ],
+            # latest
+            "example": {
```

Comment:
example and examples?

Reply:
made a mistake. I also noticed that ``examples`` in the [openapi specs are actually dictionaries](https://swagger.io/docs/specification/adding-examples/) ... I changed accordingly the new fixture ``model_cls_examples``